carma_cooperative_perception/include/carma_cooperative_perception/sdsm_to_detection_list_component.hpp

```diff
@@ -41,14 +42,29 @@ class SdsmToDetectionListNode : public carma_ros2_utils::CarmaLifecycleNode
       "input/sdsm", 1, [this](input_msg_shared_pointer msg_ptr) { sdsm_msg_callback(*msg_ptr); })},
     georeference_subscription_{create_subscription<std_msgs::msg::String>(
       "input/georeference", 1,
-      [this](std_msgs::msg::String::SharedPtr msg_ptr) { georeference_ = msg_ptr->data; })}
+      [this](std_msgs::msg::String::SharedPtr msg_ptr) { georeference_ = msg_ptr->data; })},
+    cdasim_clock_sub_{create_subscription<rosgraph_msgs::msg::Clock>(
+      "input/cdasim_clock", 1, [this](rosgraph_msgs::msg::Clock::ConstSharedPtr msg_ptr) {
```

Comment:
we need mapping from input/cdasim_clock to /cdascim_clock in carma's environment.launch.py file